### PR TITLE
Some tidying of code in openmap()

### PR DESCRIPTION
**Pull request by MichaelChirico**
Basically about using `match.arg` for `type` so as to ease the pain/scope for mistakes if the list of supported `type`s is changing over time.

### CODE REVIEW

File: OpenStreetMap/R/osm.R [hunk 1]
```diff
@@ -132,25 +132,18 @@ plot.osmtile <- function(x, y=NULL, add=TRUE, raster=TRUE, ...){
...
-	}
+openmap <- function(upperLeft,lowerRight,zoom=NULL,
+                    type=c("osm","osm-bw","maptoolkit-topo","waze","bing",
+                           "stamen-toner","stamen-terrain","stamen-watercolor",
+                           "osm-german","osm-wanderreitkarte","mapbox",
+                           "esri","esri-topo","nps","apple-iphoto","skobbler",
+                           "hillshade","opencyclemap","osm-transport",
+                           "osm-public-transport","osm-bbike","osm-bbike-german"),
+                    minNumTiles=9L, mergeTiles=TRUE){
+  type = if (substring(type, 1L, 4L) == 'http') type else match.arg(type)
```
Comment (ifellows) [inline OpenStreetMap/R/osm.R]: use <- for assignment and check indentation.

Comment (MichaelChirico) [reply OpenStreetMap/R/osm.R]: Not sure what you mean about indentation; pushing `<-` fix now...

Comment (MichaelChirico) [reply OpenStreetMap/R/osm.R]: Shall I bump the version as well while I'm at it? `0.3.4`?

Comment (ifellows) [reply OpenStreetMap/R/osm.R]: 1. Indentation in this package is done with tabs. You are using two spaces, this leads to miss-aligned lines depending on your editor settings. for example if you view the file on github this line doesn't look right.
2. Sure, bump the version.
3. I'm happy to add you as an author, under the understanding that you'll be making other significant contributions.

Comment (MichaelChirico) [reply OpenStreetMap/R/osm.R]: Sure, in fact I'll remove myself from `DESCRIPTION` for now until I put up something non-trivial for a PR. Even then I'll start as `[ctb]` = contributor for the moment...

**Comment by MichaelChirico** [comment]
@ifellows this should be ready to merge

**Pull request merged**